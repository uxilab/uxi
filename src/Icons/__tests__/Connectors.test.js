import React from 'react';
import Connectors from '../Connectors';

describe('Icon : <Connectors />', () => {
  it('match snapshot', () => {
    expect(shallow(<Connectors />)).toMatchSnapshot();
  });
});

