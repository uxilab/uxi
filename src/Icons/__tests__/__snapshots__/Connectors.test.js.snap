// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Connectors /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 15.75 15.27"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <circle
          cx="4.88"
          cy="4.27"
          r="1.5"
        />
        <circle
          cx="8.07"
          cy="7.27"
          r="1.5"
        />
        <circle
          cx="8.07"
          cy="13.77"
          r="1.5"
        />
        <circle
          cx="8.07"
          cy="1.5"
          r="1.5"
        />
        <circle
          cx="11.26"
          cy="4.27"
          r="1.5"
        />
        <path
          d="M5.77,11V9.63A6.57,6.57,0,0,1,1.3,3.53H0A7.86,7.86,0,0,0,5.77,11Z"
        />
        <path
          d="M10.37,9.48v1.39a7.84,7.84,0,0,0,5.38-7.34h-1.3A6.56,6.56,0,0,1,10.37,9.48Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
